src/main/java/org/amahi/anywhere/service/AudioService.java

```diff
@@ -316,17 +316,17 @@ public void onAudioControlChange(AudioControlChangeEvent event) {
 
     @Subscribe
     public void onAudioControlNext(AudioControlNextEvent event) {
-        startChangedAudio(getNextAudioPosition());
+        BusProvider.getBus().post(new AudioControlChangeEvent(getNextAudioPosition()));
     }
 
     @Subscribe
     public void onAudioControlPrev(AudioControlPreviousEvent event) {
-        startChangedAudio(getPreviousAudioPosition());
+        BusProvider.getBus().post(new AudioControlChangeEvent(getPreviousAudioPosition()));
     }
 
     @Subscribe
     public void onAudioCompleted(AudioCompletedEvent event) {
-        startChangedAudio(getNextAudioPosition());
+        BusProvider.getBus().post(new AudioControlChangeEvent(getNextAudioPosition()));
```

Comment:
This creates one additional call. So now, the audio player is preparing for the next position three times.